ml3d/torch/models/randlanet.py

```diff
@@ -41,136 +19,137 @@ class RandLANet(BaseModel):
     """Class defining RandLANet, a Semantic Segmentation model.
     Based on the architecture
     https://arxiv.org/abs/1911.11236#
-
-    Reference Implementation - https://github.com/QingyongHu/RandLA-Net
     """
 
     def __init__(
             self,
             name='RandLANet',
-            k_n=16,  # KNN,
-            num_layers=4,  # Number of layers
-            num_points=4096 * 11,  # Number of input points
-            num_classes=19,  # Number of valid classes
+            num_neighbors=16,
+            num_layers=4,
+            num_points=4096 * 11,
+            num_classes=19,
             ignored_label_inds=[0],
             sub_sampling_ratio=[4, 4, 4, 4],
-            dim_input=3,  # 3 + feature_dimension.
-            dim_feature=8,
+            in_channels=3,  # 3 + feature_dimension.
```

Comment:
If I use `ml3d/configs/randlanet_s3dis.yml` and the checkpoint https://storage.googleapis.com/open3d-releases/model-zoo/randlanet_s3dis_202201071330utc.pth for semantic segmentation I get:

```
RuntimeError: Error(s) in loading state_dict for RandLANet:
	size mismatch for fc0.weight: copying a param with shape torch.Size([8, 6]) from checkpoint, the shape in current model is torch.Size([8, 3]).
```

Looks like there is a discrepancy between the checkpoint file and the model.

```suggestion
            in_channels=6,  # 6 + feature_dimension.
```

Reply:
`in_channels` is already `6` in `ml3d/configs/randlanet_s3dis.yml`.
Refer https://github.com/isl-org/Open3D-ML/blob/sanskar/randlanet/ml3d/configs/randlanet_s3dis.yml#L22
I guess you are not loading the config file properly.

Reply:
You are right somehow my config was wrong Not sure how that happened. Looks all good. Sorry for that.